VSRAD.Package/BuildTools/Errors/Parser.cs

```diff
@@ -54,33 +63,47 @@ private static Message ParseClangMessage(string header)
         }
 
         private static readonly Regex ScriptErrorRegex = new Regex(
-            @"\*(?<kind>[EW]),(?<code>[^:(]+)(?>\s\((?<file>.+):(?<line>\d+)\))?:\s(?<text>.+)", RegexOptions.Compiled);
+            @"\*(?<kind>[EW]),(?<code>[^:(]+)(?>\s\((?<file>.+)\))?:\s(?<text>.+)", RegexOptions.Compiled);
 
-        private static readonly Regex ScriptErrorLocationInTextRegex = new Regex(
-            @"(?<text>.+)\s\((?<file>.+):(?<line>\d+)\)", RegexOptions.Compiled);
+        private static readonly Regex ScriptErrorTextRegex = new Regex(
+            @"(?<text>.+)\s\((?<file>.+)\)", RegexOptions.Compiled);
 
-        private static Message ParseScriptMessage(string header)
+        private static readonly Regex ScriptErrorLocationsRegex = new Regex(
+            @"(?<file>.+):(?<line>\d+)", RegexOptions.Compiled);
+
+        private static List<Message> ParseScriptMessage(string header)
```

Comment:
1. I'd change the function name to ParseScriptMessage**s** as it returns a list now.
2. I'm not sure if it's a good idea to return `null` from a function that returns a list. I would expect to get an empty list if there are no messages found in the string.
3. I wonder if making all Parse functions return a list would make the code in `ParseStderr` (specifically the fact that regexes are applied in a specific order and only one can match a given line) clearer. I'm not completely sure it's worth it though...